crates/bevy_math/src/aspect_ratio/error.rs

```diff
@@ -0,0 +1,28 @@
+//! Contains error types related to aspect ratio calculations.
+
+/// The error type returned by [`AspectRatio::try_new`] if the provided width or height is invalid.
+///
+/// [`AspectRatio::try_new`]: crate::AspectRatio::try_new
```

Comment:
I think we can remove this line of the doc, as it seems redundant. I suggest writing more the following:

"An `Error` type for [`super::AspectRatio`] if it is provided invalid width or height values"